src/components/Navbar/NavbarForLargeScreen.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+import { Button, Dropdown, DropdownButton } from "react-bootstrap";
+import { largeScreenNavbarData } from "./../../containers/Navbar/largeScreenNavbarData.js";
+
+const NavbarForLargeScreen = () => {
+  return (
+    <Router>
+      <div className="navbar">
+        <div className="navbarLogo">
+          <Link to="/">LOGO</Link>
+        </div>
+        <div className="navbarLinks">
+          {largeScreenNavbarData.map((info) => (
+            <Link to={info.target} className={info.cName} key={info.id}>
+              {info.name}
+            </Link>
+          ))}
+          <DropdownButton id="dropdownBasicButton" title="Language">
```

Comment:
why the usage of id here ?

Reply:
@halitbatur because its from react-bootstrap. I cannot change to className. It doesnt work with className